### PR TITLE
Fix yaml/__init__.pyi

### PR DESCRIPTION
**Pull request by smessmer**
This doesn't increase type safety, but prevents errors for people using yaml in mypy `--strict` mode.

### CODE REVIEW

File: third_party/2and3/yaml/__init__.pyi [hunk 1]
```diff
@@ -10,38 +10,38 @@ from typing import Any
...
-    def __init__(cls, name, bases, kwds) -> None: ...
+    def __init__(cls: Any, name: Any, bases: Any, kwds: Any) -> None: ...
 
 class YAMLObject:
     __metaclass__ = YAMLObjectMetaclass
-    yaml_loader = ...  # type: Any
-    yaml_dumper = ...  # type: Any
-    yaml_tag = ...  # type: Any
-    yaml_flow_style = ...  # type: Any
+    yaml_loader = ...  # type
```
Comment (JelleZijlstra) [inline third_party/2and3/yaml/__init__.pyi]: what happened here?

**Comment by JelleZijlstra** [comment]
Honestly I'm not sure this is an improvement. Having no type annotations in the stub indicates we still need to do the work to investigate what the actual types are, but putting Anys all over the place just adds noise.

**Comment by gvanrossum** [comment]
No, this is not a good change. If there's a problem in strict mode caused by stubs that have no annotations, maybe you should push for a change to mypy that excludes typeshed stubs from strict mode. Or something like that. Improved precision in stubs is always welcome, but fake precision by adding explicit `Any` where there previously was no annotation at all is not.

File: third_party/2and3/yaml/__init__.pyi (hunk 1 above)
Comment (smessmer) [inline third_party/2and3/yaml/__init__.pyi]: ups, fixed

**Comment by smessmer** [comment]
ok

**Pull request closed without merging**